### PR TITLE
Separate out checking vs testing scripts

### PR DESCRIPTION
**Pull request by Rheeseyb**
Separated out a pair of `check` and `check-ci` scripts that run `tsc`, `lint` and `test`

### CODE REVIEW

File: editor/package.json [hunk 1]
```diff
@@ -12,13 +12,14 @@
   },
   "scripts": {
     "clean": "rm -rf lib/",
+    "check": "tsc --noEmit && npm run lint && npm run test",
```
Comment (balazsbajorics) [inline editor/package.json:15]: these could run lint with a mute flag, so the warnings don't pollute the output

Comment (balazsbajorics) [reply editor/package.json:15]: or maybe the `npm run lint` should already mute the warnings! the warnings IMO are useful inside your code editor, not really useful as terminal output

Comment (Rheeseyb) [reply editor/package.json:15]: Good shout. I think it's worth still having a non-mute version for when you want to explicitly lint the whole code so I've also added `lint-loud` :D

Comment (balazsbajorics) [reply editor/package.json:15]: :D

**Review by balazsbajorics — APPROVED** [review]
I  forgot to approve!

**Pull request merged**